workspaces/ballerina/trace-visualizer/src/components/WaterfallView.tsx

```
@@ -0,0 +1,879 @@
+            result.push({
+                ...span,
+                level,
+                startOffsetMs,
+                durationMs,
+                hasChildren,
+                isCollapsed
+            });
+
+            // Recurse only if not collapsed
+            if (hasChildren && !isCollapsed) {
+                children.forEach(child => processSpan(child, level + 1));
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)